easybuild/easyconfigs/s/suave/suave-20160529-foss-2020b.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'PythonBundle'
+
+name = 'suave'
+version = '20160529'
+
+homepage = 'https://github.com/dellytools/suave'
+description = """suave is an interactive web application to visualize read depth
+ratios between two samples and the structural variants of one of the samples
+(typically the "case" sample in a case/control setup such as tumor/normal
+comparison)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Flask', '1.1.2'),
+    ('h5py', '3.1.0'),
+    ('Pysam', '0.16.0.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    ('Click', '7.0', {
```

Comment:
Same here, `click` is already there in `Python`, so this could be simplified to use `PythonBundle`